tools/accuracy_checker/accuracy_checker/adapters/mask_rcnn.py

```diff
@@ -71,12 +71,23 @@ def parameters(cls):
 
         return parameters
 
+    def check_params(self):
+        detection_out = self.get_value_from_config('detection_out')
+        classes_out = self.get_value_from_config('classes_out')
+        scores_out = self.get_value_from_config('scores_out')
+        boxes_out = self.get_value_from_config('boxes_out')
```

Comment:
it can be combined with class fields filling

Reply:
Added new function for set values